api/v1alpha1/pattern_types.go

```diff
@@ -83,6 +84,11 @@ type GitConfig struct {
 
 	// Unused
 	OriginRepo string `json:"originRepo,omitempty"`
+
+	// Interval in seconds to poll for drifts between origin and target repositories. Default: 30 seconds
```

Comment:
I think we can back this off significantly

Reply:
It's set to [180 seconds](https://github.com/hybrid-cloud-patterns/patterns-operator/pull/44/files#diff-3e116eb11912723137fbf22c91bdaecf841b999dbde4ddae11da3170ec515ce9R93) to match ArgoCD's interval.